CHANGELOG.md

```diff
@@ -25,6 +25,14 @@ Decidim::Budgets::Project.find_each(&:add_to_index_as_search_resource)
 Decidim::Blogs::Post.find_each(&:add_to_index_as_search_resource)
 ```
 
+- **SSL is forced on by default**
+
+Due to [#5553](https://github.com/decidim/decidim/pull/5553), SSL is turned on by default.
+
+- **GraphiQL is no longer exposed in production**
```

Comment:
Please change this as it should be "You can choose if you want the GraphiQL API to be exposed in production"

Reply:
GraphiQL is not the API. Is an editor for the GraphQL API (note the **i**). Maybe better a text like:

You can choose if you want the GraphiQL IDE editor to be exposed in production